### ISSUE TITLE
Where is the code?

### ISSUE THREAD

**Issue by ldez** [rhiokim/haroopad#442]
The code in the Github repository is the version 0.3.3, is that the code is now in another repository?
Haroopad is not open source?

**Comment by seedotlee** [comment]
There is the reason in README.

Why not open the source?

Of course, Haroopad will be the open-source at v1.0.

When Haroopad v0.1, anyone not interested. But now Haroopad has a lot of interest. It is highly appreciated.

And also I think the value of open source is really important and big for the people. For that being true, focus and commitment is very important in the early project. But it is often hampered by the open-source work.

So I just want to contain my thoughts about human writing and markdown to version 1.0.

Current version v0.13. And v0.14 will be version v1.0.

**Comment by ldez** [comment]
Note README update after my question : https://github.com/rhiokim/haroopad/commit/a0c8d6aaaa90b3bb1eb23aff0a4b609ebb2ba98f

**Comment by kemelzaidan** [comment]
Don't you think it something natural that people starts to get more interested on a certains software as it gets better? So Haroopad is not open source anymore?

**Comment by yurenchen000** [comment]
waiting for v1.0

**Comment by stefan-niedermann** [comment]
yeah. same question here. i really can't understand you. the reason in the readme.md is very flawy.

> For that being true, focus and commitment is very important in the early project. But it is often hampered by the open-source work.

You don't have to listen to the people. you can still focus on the project. but a closed-source binary is per definition **not trustable**. i want to be able to build the application from source code so i know what's going on.

Sorry, but without the source code, this application is not usable.

**Comment by ghost** [comment]
And with no source, can't use this on *BSD at all. For Linux only available as binary or Debian, which limits there too (I've had issues running pre-built binaries on Linux in the past if not built for specific distro). I'm sure interest would grow quicker if source was available to build on different systems now.

For me, by the time source is out, I'll have settled on something else.
